main.py

```diff
@@ -2,48 +2,62 @@
 # Encoding: Utf-8
 import random
 from collections import OrderedDict, deque
-import keys  # create a keys.py file with your twitter tokens if you want to run your own instance !
+# create a keys.py file with your twitter tokens if you want to run your own
+# instance !
+import keys
 import logging
 import json
 from TwitterAPI import TwitterAPI, TwitterRequestError
 
 logging.basicConfig(filename="/tmp/coffebot.log", level=logging.INFO)
 
-coffeeFile  = open("coffeeType.json")
+coffeeFile = open("coffeeType.json")
 coffeeTypes = json.load(coffeeFile)
-intro       = coffeeTypes['intro']
-multi       = coffeeTypes['multi']
-size        = coffeeTypes['size']
-coffee      = coffeeTypes['coffee']
-attribute   = coffeeTypes['attribute']
-syrup_type  = coffeeTypes['syrup_type']
-syrup       = coffeeTypes['syrup']
+intro = coffeeTypes['intro']
+multi = coffeeTypes['multi']
+size = coffeeTypes['size']
+coffee = coffeeTypes['coffee']
+attribute = coffeeTypes['attribute']
+syrup_type = coffeeTypes['syrup_type']
+syrup = coffeeTypes['syrup']
 appendition = coffeeTypes['appendition']
 
+
 def order():
+    ''' () -> str
+    Creates random order of coffee with random multi,
+    size, coffee, attribute, syrup_type, and syrup.
+    '''
     order = OrderedDict()
     order[random.choice(multi)] = True
-    for i in range(random.randint(0,5)):
+    for i in range(random.randint(0, 5)):
         order[random.choice(attribute)] = True
     order[random.choice(size)] = True
     order[random.choice(coffee)] = True
     order[random.choice(syrup_type)] = True
     order[random.choice(syrup)] = True
-    for i in range(random.randint(0,2)):
+    for i in range(random.randint(0, 2)):
         order[random.choice(appendition)] = True
     return " ".join(" ".join(order.keys()).split())
 
+
 def make_tweet(username):
+    ''' (str) -> str
+    Given the twitter username of a twitter user, returns a tweet
+    recommending user with a new coffee order.
+    '''
     while True:
         a, b = random.choice(intro)
         o = u"@"+username+" "+a+" "+order()+" "+b
```

Comment:
Multiple string concatenations isn't really best practice. You could use something like this instead:

`u'@{} {} {} {}'.format(username, a, order(), b)`